chapters/extend/driver.rst

```diff
@@ -1,4 +1,4 @@
-How to Add New Drivers
+Add New Drivers
```

Comment:
I think this should be a noun? 

I just checked https://pandas.pydata.org/docs/user_guide/text.html and it's work**ing** with, so in our case, it would be adding new drivers. What do you think?

Reply:
Done for all three